.github/actions/setup-win/action.yml

```diff
@@ -29,7 +29,18 @@ runs:
     - name: Enable long paths on Windows
       shell: powershell
       run: |
-        Set-ItemProperty -Path "HKLM:\\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1
+        if(-not (Test-Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem")) {
+          New-ItemProperty -Path "HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1 -PropertyType DWORD -Force
+        } else {
+          Set-ItemProperty -Path "HKLM:\\SYSTEM\CurrentControlSet\Control\FileSystem" -Name "LongPathsEnabled" -Value 1
+        }
+
+    - name: Enable git long paths and symlinks on Windows and disable fsmonitor daemon
```

Comment:
You don't need to do it here as these step has already been done in `_win_build` and `_win_test` workflows earlier.  The comment there tells the reason.   Basically, git config needs to be done before checking out PyTorch

Reply:
https://github.com/pytorch/pytorch/blob/main/.github/workflows/_win-build.yml#L75